### PR TITLE
[NEW] Homepage new block

### PR DESCRIPTION
**Pull request by guijun13**
## Proposed changes (including videos or screenshots)

- Create a new admin homepage block that includes:
    - Button to go to /admin/Layout setting
    - Button to show and hide the custom content to the user (a badge on top indicates if the custom content is hidden or not)
    - Button to show and hide the other blocks (add users, create channel) on the homepage (EE only button)
    
New homepage block admin view:
Screen Shot 2023-02-10 at 14 28 40

Normal user view:
Screen Shot 2023-02-10 at 17 12 39

 - Changes on the /admin/Layout settings:
     - text and homepage settings small changes
     - add Layout_Home_Custom_Block_Visible setting to show or hide custom content
     
Admin Homepage Content settings page:

Screen Shot 2023-02-10 at 15 29 18

## Further comments

TC-81

### CODE REVIEW

**Review by yash-rajpal — CHANGES_REQUESTED** [review]

**Review by yash-rajpal — CHANGES_REQUESTED** [review]
We should limit the usage of `data-qa` attributes, can you please try e2e tests using aria-label or role based selectors?

**Review by yash-rajpal — DISMISSED** [review]

**Comment by ggazzo** [comment]
could you update the description with some image?

**Review by yash-rajpal — APPROVED** [review]

**Review by yash-rajpal — APPROVED** [review]

File: apps/meteor/client/views/home/cards/CustomCard.tsx [hunk 1]
```diff
@@ -0,0 +1,92 @@
+import { Box, Button, Icon, Tag } from '@rocket.chat/fuselage';
+import { Card } from '@rocket.chat/ui-client';
+import { useRole, useSettingSetValue, useSetting, useToastMessageDispatch, useTranslation } from '@rocket.chat/ui-contexts';
+import type { ReactElement } from 'react';
+import React from 'react';
+
+import { useIsEnterprise } from '../../../hooks/useIsEnterprise';
+import CustomHomepageContent from '../CustomHomePageContent';
+
+const CustomCard = (): ReactElement | null => {
```
Comment (ggazzo) [inline apps/meteor/client/views/home/cards/CustomCard.tsx:10]: CustomCard is such a bad name for this case, CustomizationCardMaybe?

File: apps/meteor/tests/e2e/homepage.spec.ts [hunk 2]
```diff
@@ -36,12 +37,79 @@ test.describe.serial('homepage', () => {
...
 		});
 
-		test.afterAll(async () => {
+		test.describe('custom body with empty custom content', () => {
+			test.beforeAll(async ({ api }) => {
+				expect((await api.post('/settings/Layout_Home_Body', { value: '' })).status()).toBe(200);
+			});
+
+			test('expect default value in custom body', async () => {
+				await expect(adminPage.locator('[data-qa-id="homepage-custom-content-body"]')).toContainText(
```
Comment (ggazzo) [inline apps/meteor/tests/e2e/homepage.spec.ts:46]: try to avoid data-qa- props

File: apps/meteor/tests/e2e/homepage.spec.ts [hunk 3]
```diff
@@ -104,11 +172,18 @@ test.describe.serial('homepage', () => {
 			});
 		});
 
-		test.describe('custom body', () => {
+		test.describe('custom body with empty content', () => {
+			test('expect to not show custom content card', async () => {
+				await expect(regularUserPage.locator('[data-qa-id="homepage-custom-content-body"]')).not.toBeVisible();
```
Comment (ggazzo) [inline apps/meteor/tests/e2e/homepage.spec.ts:177]: same look that: homepage-custom-content-body why not just a css selector so? or even better a role selector?

**Review by ggazzo — CHANGES_REQUESTED** [review]

File: apps/meteor/tests/e2e/homepage.spec.ts (hunk 2 above)
Comment (yash-rajpal) [reply apps/meteor/tests/e2e/homepage.spec.ts]: Done!

File: apps/meteor/tests/e2e/homepage.spec.ts (hunk 3 above)
Comment (yash-rajpal) [reply apps/meteor/tests/e2e/homepage.spec.ts]: Done!

**Review by yash-rajpal — APPROVED** [review]

**Pull request merged**